index.md

```diff
@@ -2,27 +2,32 @@
 
 <div class="announcementsection">
 <h1>Announcements</h1>
-• 6/13/2019 - Zowe v1.3.0 is now available. See What's New.<br>
-• 6/1/2019 - All builds prior to Zowe v1.0.0 had been removed at May 31, 2019.
+• 6/14/2019 - Zowe v1.3.0 is now available. See <a href="https://zowe.github.io/docs-site/latest/getting-started/summaryofchanges.html#version-1-3-0-june-2019">What's New</a>.<br>
+  • 6/2/2019 - <b>All builds prior to Zowe v1.0.0</b> had been removed at May 31, 2019.
 </div>
 
 <section class="whitebackground">
 
-<h1 id="what-is-zowe">WHAT IS ZOWE?</h1>
+<h1 id="what-is-zowe">What is Zowe?</h1>
 
 <div class="section1col1">
+<p>
+Zowe is an open source project created to host technologies that benefit the Z platform from all members of the Z community (Integrated Software Vendors, System Integrators and z/OS consumers). Zowe, like Mac OS or Windows, comes with a set of APIs and OS capabilities that applications build on and also includes some applications out of the box.
+</p>
+  
 <p>
 Zowe offers modern interfaces to interact with z/OS and allows you to work with z/OS in a way that is similar to what you experience on cloud platforms today. You can use these interfaces as delivered or through plug-ins and extensions that are created by clients or third-party vendors.
 </p>
-Zowe consists of the following main components.
 
-<p><b>zLUX:</b> Contains a Web user interface (UI) that provides a full screen interactive experience. The Web UI includes many interactions that exist in 3270 terminals and web interfaces such as IBM z/OSMF. </p>
+<p>Zowe consists of the following main components.</p>
 
-<p><b>Explorer server:</b> Provides a range of APIs for the management of jobs, data sets, z/OS UNIX System Services files, and persistent data. </p>
+<p><b>Zowe Application Framework:</b> A web user interface (UI) that provides a virtual desktop containing a number of apps allowing access to z/OS function.  Base Zowe includes apps for traditional access such as a 3270 terminal and a VT Terminal, as well as an editor and explorers for working with JES, MVS Data Sets and Unix System Services.</p>
 
-<p><b>API Mediation Layer:</b> Provides an API abstraction layer through which APIs can be discovered, catalogued, and presented uniformly. </p>
+<p><b>z/OS Services:</b> Provides a range of APIs for the management of z/OS JES jobs and MVS data set services.</p>
```

Comment:
My only concern here is that the other items are matching to Zowe sub-projects, but this one isn't. Not sure....

Reply:
Hi John, I removed the z/OS Services from the component list. It is documented in the doc site as one of the four components. However I noticed that it's not on zowe.org now. I guess someone removed it at some time for certain reasons. I'll double confirm with folks to see if this is required. Thanks!